game/src/engine/physics/PhysicsService.cpp

```diff
@@ -103,6 +104,33 @@ void PhysicsService::UpdateSphereLocation(physx::PxRigidDynamic* dynamic,
     }
 }
 
+void PhysicsService::CreateRaycastFromOrigin(InputService input_service,
+                                             glm::vec3 origin,
+                                             glm::vec3 unit_dir)
+{
```

Comment:
Shouldn't need to take an argument of input_service, PhysicsService shouldn't be checking for if a button is pressed. That can be moved to a component instead to separate the logic out and make this more reusable